controllers/haproxyloadbalancer_controller.go

```diff
@@ -414,12 +414,25 @@ func (r haproxylbReconciler) reconcileLoadBalancerConfiguration(ctx *context.HAP
 		return errors.Wrap(err, "Failed to get HAProxy dataplane global config")
 	}
 
+	// Attempt to close existing transactions in case controller had previously crashed
+	if err := r.closeAllTransactions(ctx, client); err != nil {
```

Comment:
doing this here might also close on-going transactions of other `HAProxyLoadbalancers` nope ?

Reply:
Only on that same VM instance. Given that we're currently string templating everything anyway, other controllers are going to have problems reconfiguring the LB anyway. 

We could revisit this when we go to Dataplane API v2 and persist a transaction ID back to the resource to deal with re-entrancy.

Reply:
@randomvariable - can you add a `TODO` here ?

Reply:
Will do

Reply:
Sorry for the lateness. Updated, and then will follow up with a backport PR to release-0.6